content/components/relative-time.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Relative time
+status: Experimental
+---
+
+import {Box, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A relative time displays the time difference between the current time and the specified date-time.
+</Box>
+
+## Content
+
+### Past dates
+
+For past dates, the relative time outputs different formats depending on the time difference.
+
+Within the current month, the output shows a relative date in the format of `X seconds/minutes/hours/days ago`. If the time difference is less than 1 minute, it shows `just now`.
+
+If the date is in past months, the format `on D MMM` is used. If the date is in the past years, the format `on D MMM, YYYY` is used.
```

Comment:
Again here, it would benefit us to use [`DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) and it might be good to call it out as such.